packages/zowe-explorer/src/utils/LoggerUtils.ts

```diff
@@ -21,15 +24,138 @@ nls.config({
+        }
+    }
+
+    private static createMessage(msg: string, level: string): string {
+        return `[${getDate()} ${getTime()}] [${level}] ${msg}`;
+    }
+
+    private static async compareCliLogSetting(): Promise<void> {
+        const cliLogSetting = this.getZoweLogEnVar();
+        const zeLogSetting = this.zeLogLevel ?? (await this.getLogSetting());
+        if (cliLogSetting && +MessageSeverity[zeLogSetting] !== +MessageSeverity[cliLogSetting]) {
+            const notified = await this.getCliLoggerSetting();
+            if (!notified) {
+                await this.updateVscLoggerSetting(cliLogSetting);
+            }
         }
     }
+
+    private static async updateVscLoggerSetting(cliSetting: string): Promise<void> {
+        const updateLoggerButton = localize("ZoweLogger.updateLoggerSetting.update", "Update");
+        const message = localize(
+            "ZoweLogger.updateLoggerSetting.message",
+            // eslint-disable-next-line max-len
+            "Zowe Explorer now has a VS Code logger with a default log level of INFO.\nIt looks like the Zowe CLI's ZOWE_APP_LOG_LEVEL={0}.\nWould you like Zowe Explorer to update to the the same log level?",
+            cliSetting
+        );
+        await Gui.infoMessage(message, {
+            items: [updateLoggerButton],
+            vsCodeOpts: { modal: true },
+        }).then(async (selection) => {
+            if (selection === updateLoggerButton) {
+                await this.setLogSetting(cliSetting);
+            }
+            await this.setCliLoggerSetting(true);
+        });
+    }
+
+    private static async getLogSetting(): Promise<string> {
+        this.zeLogLevel = await vscode.workspace.getConfiguration().get("zowe.logger");
+        return this.zeLogLevel ?? this.defaultLogLevel;
+    }
+
+    private static async setLogSetting(setting: string): Promise<void> {
+        await SettingsConfig.setDirectValue("zowe.logger", setting, vscode.ConfigurationTarget.Global);
+    }
+
+    private static getZoweLogEnVar(): string {
+        return process.env.ZOWE_APP_LOG_LEVEL;
+    }
+
+    private static async getCliLoggerSetting(): Promise<boolean> {
+        const notified: boolean = await vscode.workspace.getConfiguration().get("zowe.cliLoggerSetting.presented");
+        return notified ? notified : false;
+    }
+
+    private static async setCliLoggerSetting(setting: boolean): Promise<void> {
+        await SettingsConfig.setDirectValue("zowe.cliLoggerSetting.presented", setting, vscode.ConfigurationTarget.Global);
+    }
+}
+
+export function getDate(): string {
+    const dateObj = new Date(Date.now());
+    const day = ("0" + dateObj.getDate()).slice(-2);
+    const month = ("0" + (dateObj.getMonth() + 1)).slice(-2);
+    return `${dateObj.getFullYear()}/${month}/${day}`;
+}
+
+export function getTime(): string {
+    const dateObj = new Date(Date.now());
+    return `${dateObj.getHours()}:${dateObj.getMinutes()}:${dateObj.getSeconds()}`;
```

Comment:
could we pad hours, minutes, and seconds with `0` in case they are `< 10` 
![image](https://user-images.githubusercontent.com/37381190/229826871-c1cd8e30-9d26-4df2-b551-4cfb3e9bf6c7.png)

![image](https://user-images.githubusercontent.com/37381190/229826926-e39b1512-d88c-4721-86a6-485ddf8d8bf4.png)

Reply:
addressed in commit https://github.com/zowe/vscode-extension-for-zowe/pull/2192/commits/560e9a4388ba8650d6b25910382cda867eb890f6